.gitignore

```diff
@@ -8,4 +8,5 @@ out
 AssemblyInfo.fs
 .DS_Store
 release.cmd
-analyzers
\ No newline at end of file
+analyzers
+.ionide
```

Comment:
with the most recent Ionide this shouldn't be required anymore - does the `.ionide` directory spawn with Ionide 6? (you'll have to delete it if it existed beforehand)

Reply:
It doesn't, that was added via extension suggestion.
> You are missing entries in your .gitignore for Ionide-specific data files. Would you like to add them?

Reply:
oh lol, guess I can update that check in ionide itself now that it's no longer necessary.